main.go

```diff
@@ -20,6 +20,7 @@ import (
 	"strings"
 	"sync"
 	"time"
+	"regexp"
```

Comment:
Before getting into the design (using a template with {netname}, etc.), I want to go over some general Go rules to follow.

Here we sort the imports, but keep the std lib ones separated from the remote ones by a new line.